UrlManager.php

```diff
@@ -356,7 +360,7 @@ protected function redirectToLanguage($language)
 
         Yii::$app->getResponse()->redirect($redirectUrl);
         if (YII_ENV_TEST) {
-            throw new \yii\base\Exception($redirectUrl);
+            throw new \yii\base\Exception(\yii\helpers\Url::to($redirectUrl));
```

Comment:
this is for correct test assert, because redirect call this function and change the result url